src/components/button/button.types.ts

```diff
@@ -15,7 +15,12 @@ export interface Button {
     icon?: string;
 
     /**
-     * True if the button should be selected. only first button will be selected.
+     * True if the button should be selected. Only first button will be selected.
      */
     selected?: boolean;
+
+    /**
+     * The label displayed in the badge
+     */
+    badge?: number | string;
```

Comment:
If this is the correct way to do it, this change should be a separate `feat` commit. But I'm quite unsure about this part of the change. Could we perhaps do something similar to this one here?
https://github.com/Lundalogik/lime-elements/blob/bf484b2277cd29ba7e62b41ed16c373fb6af43e5/src/components/menu-list/menu-list-renderer.tsx#L139

Reply:
the above also renders a badge inside the list items that are displayed in a menu

Reply:
> the above also renders a badge inside the list items that are displayed in a menu

What do you mean? @Kiarokh

Reply:
I'm also a bit unsure how to change this

Reply:
> > the above also renders a badge inside the list items that are displayed in a menu
> 
> What do you mean? @Kiarokh

I mean my suggested way of doing it also is for rendering a badge in the list items.
<img width="260" alt="image" src="https://user-images.githubusercontent.com/35954987/227219610-02873135-f6a0-4070-8209-df09fcceab78.png">
https://lundalogik.github.io/lime-elements/versions/next/#/component/limel-menu/

Reply:
I will take a look at this today

Reply:
> > > the above also renders a badge inside the list items that are displayed in a menu
> > 
> > 
> > What do you mean? @Kiarokh
> 
> I mean my suggested way of doing it also is for rendering a badge in the list items. <img alt="image" width="260" src="https://user-images.githubusercontent.com/35954987/227219610-02873135-f6a0-4070-8209-df09fcceab78.png"> [lundalogik.github.io/lime-elements/versions/next/#/component/limel-menu](https://lundalogik.github.io/lime-elements/versions/next/#/component/limel-menu/)

I still don't understand what your suggested way of doing it is? Unless I'm completely missing something here, the `Button` type in this case corresponds to the `MenuItem` type in limel-menu, and the `MenuItem` type has a `.badge` property, just like what @alexwero has added here. So I still don't understand what it is you want to change or how? 😊

Maybe we can huddle about this on Monday?

Reply:
I see. Ok. Sorry, ignore what I wrote. 

But perhaps this should be a separate `feat` commit, since button type is used more universally by other components too?

Reply:
Hmm… yeah, since this is no longer a group of normal limel-buttons (which is what this component started out as), it's perhaps a little bit weird that we still use the same type for these buttons as for limel-buttons… 🤔

How about `feat(button-group): add badge property to button type'? It's not perfect, but perhaps good enough?

Reply:
I split the commit in two